bundles/admin/admin-layereditor/view/AdminLayerForm/VisualizationTabPane/VectorStyle.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Fragment, useState } from 'react';
+import PropTypes from 'prop-types';
+import { StyleSelect } from './VectorStyle/VectorStyleSelect';
+import { Controller } from 'oskari-ui/util';
+import { Button, Message, Modal, TextInput } from 'oskari-ui';
+import { PlusOutlined } from '@ant-design/icons';
+import { OSKARI_BLANK_STYLE, StyleEditor } from 'oskari-ui/components/StyleEditor';
+
+export const VectorStyle = (props) => {
+    const [editorState, setEditorState] = useState({
+        modalVisibility: false,
+        currentStyle: OSKARI_BLANK_STYLE,
```

Comment:
No need to initialize BLANK_STYLE here now that StyleEditor can handle a missing style.

Reply:
Style initialization done differently now.